docs/providers/snowflake/sensors/snowflake.rst

```diff
@@ -0,0 +1,12 @@
+Snowflake Sensor Async
+""""""""""""""""""""""""
+
+
+Runs a sql statement repeatedly until a criteria is met in a Snowflake database. It will keep trying until success or failure criteria are met, or if the first cell is not in (0, ‘0’, ‘’, None). Optional success and failure callables are called with the first cell returned as the argument. If success callable is defined the sensor will keep retrying until the criteria is met. If failure callable is defined and the criteria is met the sensor will raise AirflowException. Failure criteria is evaluated before success criteria. A fail_on_empty boolean can also be passed to the sensor in which case it will fail if no rows have been returned.
```

Comment:
Can you elaborate on `first cell is not in (0, '0', '', None)`

Reply:
The wording follows the OSS SqlSensor documentation. I elaborated a bit on what the first cell refers to.